backend/migrating/v2/management/commands/migrate_to_v2.py

```diff
@@ -0,0 +1,442 @@
+import json
+    import logging
```

Comment:
@muhammad-ali-e seems like the indentation is off, please correct this

Reply:
Missed it

Reply:
updated